templates/websignin-form.php

```diff
@@ -1,24 +1,24 @@
 <?php
-$errors = new WP_Error();
+$login_errors = new WP_Error();
 login_header(
 	__( 'Sign in with your website', 'indieauth' ),
 	'',
-	$errors
+	$login_errors
 );
 ?>
-<form name="loginform" id="loginform" action="<?php add_query_arg( 'action', 'websignin', wp_login_url() ); ?>" method="post">
+<form name="loginform" id="loginform" action="<?php esc_url( add_query_arg( 'action', 'websignin', wp_login_url() ) ); ?>" method="post">
```

Comment:
This would output nothing, so an echo is missing:
```suggestion
<form name="loginform" id="loginform" action="<?php echo esc_url( add_query_arg( 'action', 'websignin', wp_login_url() ) ); ?>" method="post">
```